vimeo-networking/src/main/java/com/vimeo/networking/AccountStore.java

```diff
@@ -24,25 +24,48 @@
 
 import com.vimeo.networking.model.VimeoAccount;
 
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
 /**
  * Interface responsible for handling the creation, deletion, and loading of Vimeo accounts on the client.
  * <p/>
  * Created by hanssena on 4/27/15.
  */
 public interface AccountStore {
 
+    /**
+     * Load the previously saved {@link VimeoAccount} and return it
+     *
+     * @return null if no VimeoAccount could be loaded
+     */
+    @Nullable
     VimeoAccount loadAccount();
 
     /**
-     * @deprecated use {@link #saveAccount(VimeoAccount, String)} instead
-     * <p/>
-     * We find no use in storing the password when you can persist the {@link VimeoAccount} across
-     * application sessions.
+     * Saves a {@link VimeoAccount} for loading later.
+     * <p>
+     * Use this method for saving authenticated users that have an account name.
+     *
+     * @param vimeoAccount the VimeoAccount to save, should not be null when calling this method
+     * @param accountName  the <i>name</i> of the account that helps uniquely identify it in account
+     *                     stores, such as the Android AccountManager
      */
-    @Deprecated
-    void saveAccount(VimeoAccount vimeoAccount, String email, String password);
+    void saveAuthenticatedUserAccount(@NotNull VimeoAccount vimeoAccount, @NotNull String accountName);
```

Comment:
Nit: In the test class above the methods are listed in one order and they're listed in a different order in this interface. Use consistent ordering?